amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
Yes, it's also a problem w/ SSA. It assumes that all instructions have exactly one return value. My idea is to make Statement an enum with one variant for RREIL instructions and two for calls"
```rust
enum Statement {
  Simple{
    op: Operation,
    assignee: Lvalue,
  },
  ResolvedCall{
    reads: Vec<Rvalue>,
    writes: Vec<Lvalue>,
    function: Uuid,
  }
  UnresolvedCall{
    target: Rvalue,
  }
}
```
The number of calls per function isn't that large, so I hope the impact won't eat up the performance gains of the binary encoding.